src/libraries/System.Runtime.Serialization.Schema/tests/System.Runtime.Serialization.Schema.Tests.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
+    <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="System\Runtime\Serialization\Schema\Import.cs" />
+    <Compile Include="System\Runtime\Serialization\Schema\Contracts.cs" />
+    <Compile Include="System\Runtime\Serialization\Schema\Export.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="Expected.ImportedType.cs">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\src\System.Runtime.Serialization.Schema.csproj" />
+
+    <!-- NOTE TODO smolloy - Is this the right way to include CodeDom? Probably not. Feels like it should be a nuget dependency somehow.
+    But we want to bring in the package from this repo, not just any package.
+    Let's just do it this way for now and revisit once we've got all the rest of the heavy lifting done.
+    -->
+    <ProjectReference Include="$(LibrariesProjectRoot)System.CodeDom\src\System.CodeDom.csproj" />
```

Comment:
```suggestion
```

You don't need any of that as the ProjectReference that includes the source project already flows the source project's dependencies with it.